packages/xod-client/src/user/constants.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
+export const LOGIN_ERRORS = {
```

Comment:
Better to move it into `messages.js` (we're trying to move all messages for users in files `messages.js` to provide easy translation/grammar fixes by our contributors in the future).
Also, it could be separate constants and `eslint-disable` comment could be removed.

Reply:
Done!